trunner/target/emulated.py

```diff
@@ -115,6 +115,8 @@ class AARCH64A53ZynqmpQemuTarget(QemuTarget):
 
     def __init__(self):
         super().__init__("aarch64a53-zynqmp-qemu.sh")
+        # System initialization may take 30+ seconds 
```

Comment:
This trailing whitespace is not resolved:
<img width="818" height="200" alt="Screenshot from 2025-09-04 10-01-14" src="https://github.com/user-attachments/assets/c8d42911-53b3-4a0c-84f1-898adccd264b" />

Reply:
As you can see CI is failing because of that.